### PR TITLE
Add exclude rule filter in scanner results admin

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/13361

---

We decided to fix the issue above by excluding a single rule at a time (for now).

### CODE REVIEW

**Comment by willdurand** [comment]
Also requesting @diox's review because we discussed this issue during today's meeting.

File: src/olympia/scanners/admin.py [hunk 1]
```diff
@@ -116,6 +116,37 @@ def field_choices(self, field, request, model_admin):
         ]
 
 
+class ExcludeMatchedRuleFilter(SimpleListFilter):
```
Comment (diox) [inline src/olympia/scanners/admin.py]: I would have expected you to inherit from `ScannerRuleListFilter` which takes care of the choices/lookups for you, you just have to customize the queryset. Did you look into that ? Does that not work ?

Comment (willdurand) [reply src/olympia/scanners/admin.py]: I wasn't able to customize the default entry. With this patch, it looks like this (and that will, hopefully, avoid confusions):

Screen Shot 2020-03-24 at 17 16 43

Comment (diox) [reply src/olympia/scanners/admin.py]: Ok

**Pull request merged**